workshop-content/03-project-work/03.4-sns-and-sqs-setup/03.4-sns-and-sqs-setup.md

```diff
@@ -64,6 +82,9 @@ resources: # CloudFormation template syntax
             Endpoint:
               Fn::GetAtt: [<QueueName>, Arn]
             Protocol: sqs
+          -
+            Endpoint:'+6146443956784'  # Replace with your phone number
```

Comment:
Anyone reviewing this if you can serverless deploy https://github.com/AshwaryaSmridhi/simple-lambda-alert-notification repo that would be really great. Any error triggered should send an SMS but my AWS account is unable to "verify" phone numbers. So I haven't seen any SMS come through.

Reply:
is that a real number?

Reply:
No it is not.